### ISSUE TITLE
Thanks, and a q

### ISSUE THREAD

**Issue by tortfeaser** [Tobi177/venus-chargerybms#8]
This looks like just the software I need for my Victron system, but I've got a question. Will this software on VenusOS tell VE.Bus connected chargers to reduce charge current when overvoltage is imminent? Or is that something I tell VenusUS to do based on chargerybms data?

**Comment by Spidey4fun** [comment]
> This looks like just the software I need for my Victron system, but I've got a question. Will this software on VenusOS tell VE.Bus connected chargers to reduce charge current when overvoltage is imminent? Or is that something I tell VenusUS to do based on chargerybms data?

Did you discover the answer to your question?   Im curious also.

**Comment by Tobi177** [comment]
No, the chargery driver does not control chargers and their charge current. The driver only reports the chargery data to the victron system. You cannot use the chargery data to reduce the charge current on over-voltage. The software does not control anything.

**Comment by Spidey4fun** [comment]
OK.  So the data from the chargery is for information only.  Got it.  Thanks.
